ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayloadTest.java

```diff
@@ -198,6 +199,7 @@ public void shouldReturnSuccessOnAlreadyPresent() {
 
   @Test
   public void shouldReturnInvalidOnBadTerminalBlock() {
+    if (!validateTerminalPoWBlock()) return;
```

Comment:
For this test, probably better handled by cut-and-pasting this whole test into V1Test

Reply:
Yes, updated